source/Calamari.Common/Plumbing/Deployment/PackageRetention/PackageIdentity.cs

```diff
@@ -75,5 +75,15 @@ public override string ToString()
         {
             return $"{PackageId} v{Version}";
         }
+
+        public static PackageIdentity GetPackageIdentity(IManagePackageUse journal, IVariables variables, string[] commandLineArguments)
```

Comment:
What's the `journal` and `commandLineArguments` used for here?

Reply:
Good pick-up! Those shouldn't be in this PR, they're for the next one...